thunder/tests/test_core.py

```diff
@@ -2111,28 +2111,6 @@ def func(qkv):
     assert "scaled_dot_product_attention" in tuple(bsym.sym.id for bsym in traces[-1].bound_symbols)
 
 
-@instantiate(dtypes=NOTHING)
-def test_no_passthrough_symbol(executor, device, _):
```

Comment:
On second thought about using `shallow_copy` in `Symbol.__call__`: There's no way to make this test sensible, but it seems to have been added for a reason.  I'm not sure where to find the issue(?) "backward trace contains symbols not present in forward that cause # NotImplementedError" to see what this test is really trying to guard against. @IvanYashchuk